### PR TITLE
Revise node samples pt. 1

### PR DESCRIPTION
**Pull request by derisen**
This PR revises the msal-node samples without views and automated tests. These are:

- auth-code-with-certs
- auth-code-key-vault
- on-behalf-of
- refresh-token
- username-password
- client-credentials

Changes include:
- Readme updates
- package.json fixes (install paths and etc.)
- cache files (auto create if doesn't exist)
- source code comments
- removing credentials from source code

### CODE REVIEW

File: samples/msal-node-samples/README.md [hunk 1]
```diff
@@ -2,11 +2,36 @@
...
+| [auth-code](./auth-code/README.md)                         | web app (public client)           | authorization code                  |
+| [auth-code-pkce](./auth-code-pkce/README.md)               | web app (public client)           | authorization code with PKCE        |
+| [auth-code-with-certs](./auth-code-with-certs/README.md)   | web app (confidential client)     | authorization code                  |
+| [auth-code-key-vault](./auth-code-key-vault/README.md)     | web app (confidential client)     | authorization code                  |
+| [silent-flow](./silent-flow/README.md)                     | web app (confidential client)     | authorization code                  |
+| [on-behalf-of](./on-behalf-of/README.md)                   | web app (confidential client)     | on-behalf-of                        |
+| [refresh-token](./refresh-token/README.md)                 | web app (confidential client)     | refresh token                       |
+| [username-password](./username-password/README.md)         | console app (public client)       | resource owner password credentials |
```
Comment (sameerag) [inline samples/msal-node-samples/README.md:18]: lets also add `console/daemon` app?

Comment (derisen) [reply samples/msal-node-samples/README.md]: ah, of course -done!

**Comment by jasonnutter** [comment]
Can you also make sure this list is up to date? https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node#samples

File: samples/msal-node-samples/client-credentials/index.js [hunk 2]
```diff
@@ -29,8 +29,8 @@ const cca = new msal.ConfidentialClientApplication(config);
 // The scope is always in the format "<resource>/.default"
 const clientCredentialRequest = {
     scopes: ["https://graph.microsoft.com/.default"],
-    azureRegion: "westus2",
-    skipCache: true, 
+    azureRegion: "REGION_NAME", // (optional) specify the region you will deploy your application to here (e.g. "westus2")
```
Comment (derisen) [inline samples/msal-node-samples/client-credentials/index.js:32]: @sameerag added this back and pointed to docs in readme

File: samples/msal-node-samples/username-password-cca/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# MSAL Node Standalone Sample: Resource Owner Password Credentials (ROPC) Grant
+
+This sample demonstrates how to implement an MSAL Node [confidential client application](../../../lib/msal-node/docs/initialize-confidential-client-application.md) to sign in a user and acquire an access token using the [OAuth 2.0 ROPC grant](https://datatracker.ietf.org/doc/html/rfc6749#section-4.3). Note that this flow requires a very high degree of trust in the application, and carries risks which are not present in other flows. You should only use this flow when other more secure flows can't be used (e.g. automated testing).
+
+## Prerequisites
+
+- ROPC grant cannot be used with personal Microsoft accounts (MSA)
+- ROPC grant cannot be used if the user needs to perform multi-factor authentication
+- MSAL Node supports ROPC only when the authorization server is OpenID-compliant (e.g. ADFS 2016 is supported, but WS-Federation is not).
```
Comment (derisen) [inline samples/msal-node-samples/username-password-cca/README.md:9]: @sameerag slightly reworded

File: samples/msal-node-samples/client-credentials/index.js (hunk 2 above)
Comment (samuelkubai) [reply samples/msal-node-samples/client-credentials/index.js:32]: Makes sense to me.

File: samples/msal-node-samples/README.md [hunk 4]
```diff
@@ -100,17 +96,17 @@ const config = {
 Once you are in the sample application directory and you've configured the sample application to match your case, you can install all of the dependencies by running:
 
 ```bash
-$ npm install
+    npm install
```
Comment (samuelkubai) [inline samples/msal-node-samples/README.md:99]: @derisen should we ensure they have run `npm run build:all` for the msal-node library before running `npm install` as the msal-node library is defined as a local dependency in the package.json file

**Review by samuelkubai — APPROVED** [review]
Left a single question, but otherwise looks good to me. Great work @derisen

File: samples/msal-node-samples/README.md (hunk 4 above)
Comment (derisen) [reply samples/msal-node-samples/README.md:99]: Good idea -let me add a pointer for setting up msal-node

File: samples/msal-node-samples/username-password-cca/README.md (hunk 3 above)
Comment (sameerag) [reply samples/msal-node-samples/username-password-cca/README.md:9]: Isn't this ADFS 2019?

**Review by sameerag — APPROVED** [review]

**Comment by sameerag** [comment]
Added a comment on ADFS, otherwise lgtm. Approved.

File: samples/msal-node-samples/username-password-cca/README.md (hunk 3 above)
Comment (derisen) [reply samples/msal-node-samples/username-password-cca/README.md:9]: it is 🤦‍♂️ fixing..

**Pull request merged**